src/components/action-toolbar/action-toolbar.js

```diff
@@ -188,6 +196,7 @@ class ActionToolbar extends React.Component {
 
         <div className='carbon-action-toolbar__actions'>
           { this.actions() }
+          { this.props.actionExtras }
```

Comment:
does this need to trigger any `onClick` events with the list of selected items (like the other actions)?

does it also need to disable/enable depending on if items are selected?

Reply:
This needs more work. I believe it will need to be disabled until a table item is selected.

Reply:
You could get it to work using a pattern discussed in @SeanArmstrong's thread https://github.com/Sage/carbon/issues/1539

(I don't mean refactor the `actions` prop 😄  but the multi action button could be added using the discussed technique)

Reply:
Sorry @andrewjtait, I don't fully understand. Do you mean I should check for the onClick event here in gac?
https://github.com/Sage/sage_one_advanced/pull/12910/files#diff-4ecfd10dd8f9f7dd4e6bdf8adc95e627R287

Reply:
I not sure we can complete a 100% of the pattern described in #1539 due to the further refactoring of table so that you can provide your own actionToolbar etc.

Perhaps for now table could receive actionToolbarChildren which are added after the icons. (this is basically what you have now). The next possible step is adding some sort of wrapper component for multi action toolbar which means that when a onClick event happens to one of the buttons it has the context of what items are already selected without the implementor having to bind the information.